megamek/src/megamek/common/MechView.java

```diff
@@ -296,7 +295,14 @@ public MechView(Entity entity, boolean showDetail, boolean useAlternateCost, boo
                         .append(warningEnd());
                 }
             }
-            sBasic.add(new LabeledElement(Messages.getString("MechView.Movement"), moveString.toString())); //$NON-NLS-1$
+
+            // TODO : Add STOL message as part of the movement line
+            if (isConvFighter && ((Aero) entity).isVSTOL()) {
+                sBasic.add(new LabeledElement(Messages.getString("MechView.Movement"),
+                        moveString.toString().concat(" (VSTOL)"))); //$NON-NLS-1$
```

Comment:
Strings that are displayed to the user should go in the i18n resource file.

Reply:
How do you handle untranslated string for German or Russian? Should I do VSTOL_de, de_VSTOL, leave it out?

Reply:
Don't worry about the translations. Just put it in the default locale so if or when someone volunteers to update the translations it will be there for them.